core/schema.sql

```diff
@@ -7,6 +7,7 @@
 
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```

Comment:
this is Postgres 9.6 only. should probably delete this since some of us are still on 9.5